dura/trust_path/template/default.default.php

```diff
@@ -36,11 +36,11 @@
 
 </form>
 
-<?php if ( file_exists(DURA_TEMPLATE_PATH.'/footer.html') ) : ?>
+
 <div class="footer">
-<?php require DURA_TEMPLATE_PATH.'/footer.html' ?>
+<p><?php e(t("Durarara like chat room")) ?></p>
```

Comment:
Translation on this part will not work due to this change?
https://github.com/schnabear/drrr-like-chat/pull/3/files#diff-24afe5e39ad39abf102a6819d115c60bL50